lib/pages/chat_list/chat_list_item_subtitle.dart

```diff
@@ -60,39 +67,49 @@ class ChatListItemSubtitle extends StatelessWidget with ChatListItemMixin {
                 room.lastEvent == null) {
               return const SizedBox.shrink();
             }
-
             final isMentionned = snapshot.data!
                 .getAllMentionedUserIdsFromMessage(room)
                 .contains(Matrix.of(context).client.userID);
-            return AnimatedContainer(
-              duration: TwakeThemes.animationDuration,
-              curve: TwakeThemes.animationCurve,
-              padding: const EdgeInsets.only(bottom: 4),
-              height: ChatListItemStyle.mentionIconWidth,
-              width: isMentionned && room.isUnreadOrInvited
-                  ? ChatListItemStyle.mentionIconWidth
-                  : 0,
-              decoration: BoxDecoration(
-                color: Theme.of(context).colorScheme.primary,
-                borderRadius: BorderRadius.circular(AppConfig.borderRadius),
-              ),
-              child: Center(
-                child: isMentionned && room.isUnreadOrInvited
-                    ? Text(
-                        '@',
-                        style: TextStyle(
-                          color: isMentionned
-                              ? Theme.of(context).colorScheme.onPrimary
-                              : Theme.of(context)
-                                  .colorScheme
-                                  .onPrimaryContainer,
-                          fontSize:
-                              Theme.of(context).textTheme.labelMedium?.fontSize,
-                        ),
-                      )
-                    : Container(),
-              ),
-            );
+            return room.lastEvent?.senderId == Matrix.of(context).client.userID
+                ? Icon(
+                    Icons.done_all,
+                    color: room.lastEvent!.receipts.isEmpty
+                        ? LinagoraRefColors.material().tertiary[30]
+                        : LinagoraSysColors.material().secondary,
+                    size: 20,
```

Comment:
no hardcoded sizes, padding, margin, etc. in widgets. Please move this in style widget